streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -131,6 +131,40 @@
     void to(Serde<K> keySerde, Serde<V> valSerde, String topic);
 
     /**
+     * Print the elements of this stream to System.out
```

Comment:
We need to state that users are expected to override toString() function for these functions in JavaDoc.